src/v3/src/transformer/password/transformEnrollPasswordAuthenticator.ts

```diff
@@ -129,21 +129,18 @@ export const transformEnrollPasswordAuthenticator: IdxStepTransformer = ({
   };
 
   const userInfo = getUserInfo(transaction);
-  uischema.elements = [
```

Comment:
Overriding elements like this causes this issue where we remove API provided transformed fields

Reply:
So in the case of revoke session, are we using the base field transform to get the checkbox element to appear?

Reply:
Yessir exactly! Its gets transformed into a checkbox earlier in the flow and this just supplements the pre-baked elements with additional things like Title, etc